Doc/library/curses.rst

```diff
@@ -1279,6 +1279,18 @@ The :mod:`curses` module defines the following data members:
    A bytes object representing the current version of the module.  Also available as
    :const:`__version__`.
 
+
+.. data:: ncurses_version
+
+   A named tuple containing the three components of the ncurses library
+   version: *major*, *minor*, and *patch*.  All values are integers.  The
+   components can also be accessed by name,  so ``curses.ncurses_version[0]``
+   is equivalent to ``curses.ncurses_version.major`` and so on.  Available
```

Comment:
Please write "Available ..." in a new paragraph, to have a similar style than "Availability: ..." used in the os documentation.